### PR TITLE
Remove usage of deprecated APIs

### PR DESCRIPTION
**Pull request by saberduck**
Fixes https://github.com/SonarSource/SonarJS/issues/4641

### CODE REVIEW

File: sonar-plugin/sonar-javascript-plugin/src/main/java/org/sonar/plugins/javascript/bridge/BridgeServer.java [hunk 1]
```diff
@@ -33,7 +33,7 @@
 import org.sonarsource.api.sonarlint.SonarLintSide;
 
 @ScannerSide
-@SonarLintSide(lifespan = MULTIPLE_ANALYSES)
+@SonarLintSide(lifespan = INSTANCE)
```
Comment (ilia-kebets-sonarsource) [inline sonar-plugin/sonar-javascript-plugin/src/main/java/org/sonar/plugins/javascript/bridge/BridgeServer.java:36]: Is `INSTANCE` the replacement for the deprecated `MULTIPLE_ANALYSES`?

File: sonar-plugin/sonar-javascript-plugin/src/main/java/org/sonar/plugins/javascript/bridge/BridgeServerImpl.java [hunk 2]
```diff
@@ -177,7 +175,8 @@ void deploy(Configuration configuration) throws IOException {
   }
 
   void startServer(SensorContext context, List<Path> deployedBundles) throws IOException {
-    PROFILER.startDebug("Starting server");
+    LOG.debug("Starting server");
+    long start = System.currentTimeMillis();
```
Comment (ilia-kebets-sonarsource) [inline sonar-plugin/sonar-javascript-plugin/src/main/java/org/sonar/plugins/javascript/bridge/BridgeServerImpl.java:179]: Is there no equivalent for Profiler in slf4j?

**Review by ilia-kebets-sonarsource — APPROVED** [review]
Some questions otherwise LGTM!

File: sonar-plugin/sonar-javascript-plugin/src/main/java/org/sonar/plugins/javascript/bridge/BridgeServer.java (hunk 1 above)
Comment (saberduck) [reply sonar-plugin/sonar-javascript-plugin/src/main/java/org/sonar/plugins/javascript/bridge/BridgeServer.java:36]: yes, you can check the javadoc

File: sonar-plugin/sonar-javascript-plugin/src/main/java/org/sonar/plugins/javascript/bridge/BridgeServerImpl.java (hunk 2 above)
Comment (saberduck) [reply sonar-plugin/sonar-javascript-plugin/src/main/java/org/sonar/plugins/javascript/bridge/BridgeServerImpl.java:179]: unfortunately no, if really needed we can create something similar on our own

**Pull request merged**